.azure-pipelines/azure-pipelines-base.yml

```diff
@@ -51,22 +58,20 @@ jobs:
       python -m pip install --upgrade pip
       python -m pip install wheel
       python -m pip install --no-cache-dir -r requirements.txt
-      python -m pip install --no-cache-dir -r requirements-optional.txt
```

Comment:
Are still using this file somewhere?

Reply:
Yes, all the jobs that now have the `_optional` suffix install the requirements in this file. It is done in the next step now though because it doesn't happen for all jobs and so needs its own condition to be evaluated.